index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once 'Autor.php';
```

Comment:
```suggestion
require_once "Autor.php";
```
Usar el mismo tipo de comillas en el proyecto